xbmc/video/VideoDatabase.cpp

```diff
@@ -4566,6 +4589,13 @@ void CVideoDatabase::UpdateTables(int iVersion)
       }
     }
   }
+  if (iVersion < 94)
```

Comment:
So basically it's always one bigger then the latest update instructions go. Seems like its mostly important for the alpha users?